File: web-programming/lab-asp-net/asp_net_lab/Models/Domain/Article.cs
using System.ComponentModel.DataAnnotations;

namespace asp_net_lab.Models.Domain
{
    public class Article
    {
        public int Id { get; set; }
        public string Title { get; set; }
        [Display(Name = "Author")]
        public int AuthorId { get; set; }
        [StringLength(10000, MinimumLength = 5)]
        public string Text { get; set; }
        public DateTime Date { get; set; }
		[Display(Name = "Category")]
		public int CategoryId { get; set; }

        public Author? Author { get; set; }
        public Category? Category { get; set; }
    }
}

File: web-programming/lab-php/model/entity/article.php
<?php 
    class Article implements JsonSerializable {
        private $title;
        private $authorId;
        private $text;
        private $date;
        private $categoryId;

        public function __construct($title, $authorId, $text, $date, $categoryId) {
            $this->title = $title;
            $this->authorId = $authorId;
            $this->text = $text;
            $this->date = $date;
            $this->categoryId = $categoryId;
        }

        public function jsonSerialize() {
            return get_object_vars($this);
        }
    }
?>